dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/ConnectorArguments.java

```diff
@@ -229,6 +232,37 @@ public class ConnectorArguments extends DefaultArguments {
           .withOptionalArg()
           .describedAs("sekr1t");
 
+  private final OptionSpec<ZonedDateTime> optionStartDate =
+      parser
+          .accepts(
+              OPT_START_DATE,
+              "Inclusive start date for data to export, value will be truncated to hour")
+          .withOptionalArg()
+          .ofType(Date.class)
+          .withValuesConvertedBy(ZonedParser.withDefaultPattern(DayOffset.START_OF_DAY))
+          .describedAs("2001-01-15[ 00:00:00.[000]]");
+
+  private final OptionSpec<ZonedDateTime> optionEndDate =
+      parser
+          .accepts(
+              OPT_END_DATE,
+              "Inclusive end date for data to export, value will be truncated to hour")
```

Comment:
```suggestion
              "Exclusive end date for data to export, value will be truncated to hour")
```
Based on PR description this parameter should not be inclusive.

Reply:
Yes, thank you. fixed.